src/utils/num.ts

```diff
@@ -0,0 +1,3 @@
+export const getRandomInt = (max: number): number => {
+  return Math.floor(Math.random() * max) + 1;
```

Comment:
I feel `getRandomInt` should return an integer in [0, n-1] though - having it start from 1 seems weird to me.